File: qe-selenium-java/Activity20.java
package demos;

import java.time.Duration;
import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class Activity20 {
public static void main(String[] args) {
    WebDriver driver = new FirefoxDriver();
    WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
    driver.get("https://training-support.net/webelements/alerts");
    System.out.println("Title of the Page: "+driver.getTitle());

    System.out.println(driver.getTitle());

    driver.findElement(By.id("prompt")).click();

    wait.until(ExpectedConditions.alertIsPresent());

    Alert alert = driver.switchTo().alert();

    System.out.println("Text in Alert: "+alert.getText());

    alert.sendKeys("Awesome!");
    alert.accept();

    // Close browser
    driver.quit();
    }
}
    

File: qe-selenium-python/Activity20.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC


with webdriver.Firefox() as driver:

    wait = WebDriverWait(driver, 10)

    driver.get("https://training-support.net/webelements/alerts")

    print("Title of the Page:", driver.title)
    print(driver.title)

    driver.find_element(By.ID, "prompt").click()

    # Wait until prompt alert is present
    wait.until(EC.alert_is_present())

    alert = driver.switch_to.alert

    print("Text in Alert:", alert.text)

    alert.send_keys("Awesome!")
    alert.accept()
